### PR TITLE
Change operationIds of SignalRService from Signalr to SignalR

### PR DESCRIPTION
**Pull request by juniwang**
We used `Signalr` because the python SDK class names will look better(`signalr_XXX` instead of `signal_rXXX`). But I confirmed with Python SDK team the names are transparent to customers hence that won’t be an issue.  In this PR, we are going roll back the original `SignalR`

Tested on local follow https://github.com/Azure/adx-documentation-pr/wiki/Azure-Swagger-Tools. All validations passed.

### CODE REVIEW

**Review by sergey-shandar — APPROVED** [review]

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (lmazuel) [inline specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: You should call this one `Operations_List`. It will be consistent with everybody else:
https://github.com/Azure/azure-rest-api-specs/blob/9666a766d478b21b587210260fb7de342ae9980e/specification/storage/resource-manager/Microsoft.Storage/stable/2017-10-01/storage.json#L29

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (lmazuel) [inline specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: Why don't you call the operation ID directly "Accounts"? right now, this is redundant: we know it's signal R and gives Python code like that (but I guess it will be close in other language as well):
```python
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalRManagementClient(**parameters)
client.signal_r.create_or_update(**parameters)
```
wouldn't be better with:
```python
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalRManagementClient(**parameters)
client.accounts.create_or_update(**parameters)
```

**Review by lmazuel — CHANGES_REQUESTED** [review]
I tag "request change" just to be sure you got a chance to think about my comments, but I won't block Sergey to merge if you want to keep it.

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (lmazuel) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: Note that I said "accounts", but I mean "something with is not the 'signal_r' string"

**Review by sergey-shandar — APPROVED** [review]
@juniwang please, address issues in another PR. Let me know the number of the new PR so I can assign it to myself.

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (sergey-shandar) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: Issue #2899

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (sergey-shandar) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: Issue #2899

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (juniwang) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: updated

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (juniwang) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: We decided to using Signalr for OperationId/Schema Id for the client SDKs.

**Comment by sergey-shandar** [comment]
@lmazuel could you approve this PR? I can't merge without your approval.

File: specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json
Comment (juniwang) [reply specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json]: See PR https://github.com/Azure/azure-rest-api-specs/pull/2904. In the new version, it would look like:
```
import azure.mgmt.signalr
client = azure.mgmt.signalr.SignalrManagementClient(**parameters)
client.signalr.create_or_update(**parameters)
```

To keep the name consistent, we updated all Schema Ids such as `SignalrResource`, `SignalrManagementClient`.  I checked the generated Python/.Net SDK files, the file names and class names are expected as `Signalr`. `SignalR` will only exist in comments, descriptions and RP namespace.

**Review by lmazuel — APPROVED** [review]

**Pull request merged**
